src/applications/proxy-rewrite/proxy-rewrite-whitelist.json

```diff
@@ -64,7 +64,7 @@
     {
       "hostname": "www.hepatitis.va.gov",
       "pathnameBeginning": "/",
-      "cookieOnly": true
+      "cookieOnly": false
```

Comment:
If I understand it correctly, `cookieOnly` is used by the `proxy-rewrite` application to determine if the injected assets should be activated. 

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/proxy-rewrite/proxy-rewrite-entry.jsx#L191

Reply:
Correct, that's how I'm currently testing, using a cookie to display the new header. Once, it's turned off it will display the header without the cookie